File: java/lib/SnobotSim/SimulatorComponents/Accelerometer/AccelerometerWrapper.cpp
/*
 * AccelerometerWrapper.cpp
 *
 *  Created on: May 7, 2017
 *      Author: PJ
 */

#include "SnobotSim/SimulatorComponents/Accelerometer/AccelerometerWrapper.h"

AccelerometerWrapper::AccelerometerWrapper(const std::string& aName):
    AModuleWrapper(aName),
    mAcceleration(0)
{

}

AccelerometerWrapper::~AccelerometerWrapper()
{

}

void AccelerometerWrapper::SetAcceleration(double aAcceleration)
{
    mAcceleration = aAcceleration;
}

double AccelerometerWrapper::GetAcceleration()
{
    return mAcceleration;
}

File: snobot_sim_java/java/snobot_sim_java/com/snobot/simulator/simulator_components/accelerometer/AccelerometerWrapper.java
package com.snobot.simulator.simulator_components.accelerometer;

import com.snobot.simulator.module_wrapper.ASensorWrapper;

public class AccelerometerWrapper extends ASensorWrapper
{
    protected double mAcceleration;

    public AccelerometerWrapper(String aName)
    {
        super(aName);
    }

    public void setAcceleration(double aAcceleration)
    {
        mAcceleration = aAcceleration;
    }

    public double getAcceleration()
    {
        return mAcceleration;
    }

}
